src/uu/od/src/od.rs

```diff
@@ -286,26 +470,21 @@ impl OdOptions {
 pub fn uumain(args: impl uucore::Args) -> i32 {
     let args = args.collect_str();
 
-    let opts = create_getopts_options();
-
-    let matches = match opts.parse(&args[1..]) {
-        Ok(m) => m,
-        Err(f) => {
-            show_usage_error!("{}", f);
-            return 1;
-        }
-    };
+    let mut clap_opts = create_opts();
+    let clap_matches = clap_opts
+        .clone() // Clone to reuse clap_otps to print help
+        .get_matches_from(args.clone());
 
-    if matches.opt_present("help") {
-        println!("{}", opts.usage(&USAGE));
+    if clap_matches.is_present(options::HELP) {
```

Comment:
this isn't needed, it is managed by clap

Reply:
Yup, thought we would have issues because of the -h flag, but it works fine